engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/gateway/InclusiveGatewayTest.java

```diff
@@ -626,6 +626,50 @@ public void shouldCreateDeploymentInclusiveGatewayWithDefaultFlow() {
         .isNotNegative();
   }
 
+  @Test
+  public void shouldTaskTheDefaultSequenceFlowEvenMissingVariable() {
+    // given
+    final String processId = Strings.newRandomValidBpmnId();
+    final BpmnModelInstance processDefinition =
+        Bpmn.createExecutableProcess(processId)
+            .startEvent()
+            .inclusiveGateway("inclusive")
+            .sequenceFlowId("s1")
+            .conditionExpression("= contains(str,\"a\")")
+            .endEvent("end1")
+            .moveToLastGateway()
+            .sequenceFlowId("s2")
+            .conditionExpression("= contains(str,\"b\")")
+            .endEvent("end2")
+            .moveToLastInclusiveGateway()
+            .defaultFlow()
+            .sequenceFlowId("s3")
+            .conditionExpression("= contains(str,\"c\")")
+            .endEvent("end3")
+            .done();
+    ENGINE.deployment().withXmlResource(processDefinition).deploy();
+
+    // when
+    final long processInstance =
+        ENGINE
+            .processInstance()
+            .ofBpmnProcessId(processId)
+            .withVariable("missing_str", "d")
```

Comment:
What I meant was not adding that var, but allowing the other conditions to evaluate correctly but not fulfilling them, so we need a `str` variable (e.g. containing an empty list):
```suggestion
            .withVariable(Map.of("str", List.of()))
```